webapp/app.py

```diff
@@ -58,7 +58,8 @@ def home():
                 chapter,
                 user,
             )
-    return flask.render_template('home.html', chapters=chapters)
+        score, max = answers_module.compute_score(chapter,user)
```

Comment:
@broumec this is not valid! You only take into account the last score/max!

You can either dynamically add a field `score` to each `chapter` object (it is a bit dirty) or create a dictionary/list containing all the data about the scores.

Reply:
I changed it, can you see my modification ?

Reply:
Yes, seems great!